docs/source/azure/configuration/serverlessfunctions.rst

```diff
@@ -1,7 +1,7 @@
 .. _azure_azurefunctions:
 
-Azure Functions Support
-=======================
+Serverless - Azure Functions Support
```

Comment:
This is OK for today, but I suspect this will evolve to a variety of hosting options (we have some non-functions stuff based on Azure Container Instances coming soon) - and it also mentions the local CLI mode in here under modes - so if we wanted to make this more generic than serverless it might fit.

We can update that as we add more hosting options though...